hw/ip/gpio/data/gpio.hjson

```diff
@@ -27,10 +27,10 @@
       notes:              "",
     }
     {
-      version:            "1.1.0",
```

Comment:
Maybe undo that, given it is only an addition but not a change in behaviour.

Reply:
@rswarbrick I think that should be a `1.2.0` since it is backward compatible. What about the design and verification stage. Similar to the comment from @GregAC in https://github.com/lowRISC/opentitan/pull/24489#issuecomment-2405489959 - should we leave them as they are?

Reply:
SW register layout is different though.

Reply:
I think this should be a minor version increment because the new registers are at the end of the address space of this IP -- so existing registers don't move and existing SW continues to work. Or am I missing something?

Reply:
Yes. I drop that from the commit - this was changed on the integrated_dev branch. The comment was from before we had the versioning discussion.